packages/notebook/test/actions.spec.ts

```diff
@@ -1111,6 +1112,26 @@ describe('@jupyterlab/notebook', () => {
         expect(result).toBe(true);
         expect(widget.isSelected(widget.widgets[2])).toBe(true);
       });
+
+      it('should run the selected cells', async () => {
+        let emitted = 0;
+        let notebook: Notebook | null = null;
+        let lastCell: Cell | null = null;
+        NotebookActions.selectionExecuted.connect(async (_, args) => {
+          notebook = args[0];
+          lastCell = args[1];
```

Comment:
This currently fails on CI:

```
Error: test/actions.spec.ts(1121,22): error TS7053: Element implicitly has an 'any' type because expression of type '0' can't be used to index type '{ notebook: Notebook; lastCell: Cell<ICellModel>; }'.
  Property '0' does not exist on type '{ notebook: Notebook; lastCell: Cell<ICellModel>; }'.
```

```suggestion
          notebook = args.notebook;
          lastCell = args.lastCell;
```